sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -333,6 +334,24 @@ public enum SelectionMarkState
         Selected = 0,
         Unselected = 1,
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct TextStyle : System.IEquatable<Azure.AI.FormRecognizer.Models.TextStyle>
```

Comment:
It seems we added this struct to the public API, but where is it being used?

Reply:
Oh I missed this one. I will hide